### PR TITLE
DDF-4182 Relocated jacoco lenient limit to codice-maven

### PR DESCRIPTION
**Pull request by paouelle**
Relocated jacoco lenient limit to codice-maven and reverted to previous artifact name.

### CODE REVIEW

**Comment by paouelle** [comment]
build now

File: support-jacoco/pom.xml [hunk 1]
```diff
@@ -7,9 +7,8 @@
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>support-jacoco</artifactId>
-    <name>DDF Support Jacoco</name>
-    
+    <artifactId>coverage-updater</artifactId>
```
Comment (coyotesqrl) [inline support-jacoco/pom.xml]: ❓ Should this file be moved to a `coverage-updater` directory?

Comment (paouelle) [reply support-jacoco/pom.xml]: I simply reverted back to before we added the lenient limit class. It was defined like that. I can surely move it if we want to.

Comment (coyotesqrl) [reply support-jacoco/pom.xml]: Nah. It's fine the way it is. 👍

File: pom.xml [hunk 2]
```diff
@@ -44,6 +46,9 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
+
+        <codice-maven.version>0.1</codice-maven.version>
```
Comment (tbatie) [inline pom.xml]: Should this be `codice-test.version`?

**Pull request merged**